src/com/aca/LarisaGabrielyan/Homework1/Product.java

```diff
@@ -0,0 +1,24 @@
+package com.aca.LarisaGabrielyan.Homework1;
+
+public class Product {
+    String name;
+    Type videotype;
+    VideoQuality videoQuality;
```

Comment:
Product can have nothing related to videos so properties related video are redundant